src/main/scala/beam/utils/csv/writers/PopulationCsvWriter.scala

```diff
@@ -28,12 +28,17 @@ object PopulationCsvWriter extends ScenarioCsvWriter {
       val customAttributes: AttributesOfIndividual =
         person.getCustomAttributes.get("beam-attributes").asInstanceOf[AttributesOfIndividual]
 
-      val excludedModes = personAttributes
-        .getAttribute(person.getId.toString, "excluded-modes")
-        .toString
-        .replaceAll(",", ArrayItemSeparator)
-        .split(ArrayItemSeparator)
-        .mkString(ArrayStartString, ArrayItemSeparator, ArrayEndString)
+      // `personAttributes.getAttribute(...)` can return `null`
+      val excludedModes = Option(
+        personAttributes
+          .getAttribute(person.getId.toString, "excluded-modes")
+      ).map { attrib =>
+          attrib.toString
+            .replaceAll(",", ArrayItemSeparator)
+            .split(ArrayItemSeparator)
+            .mkString(ArrayStartString, ArrayItemSeparator, ArrayEndString)
+        }
+        .getOrElse("")
```

Comment:
Is an empty string ok though? I at least wanted to raise that, in case an exception would be better?

Reply:
the exception was there before (NPE) :-)
Agree it should at least be logged